docs/learn/learn-consensus.md

```diff
@@ -1,103 +1,48 @@
 ---
 id: learn-consensus
-title: Polkadot Consensus
-sidebar_label: Consensus
+title: Polkadot's Consensus Protocols
+sidebar_label: Consensuses
```

Comment:
Sidebar label of consensus is cleaner and implies that it covers the topic of consensus. It is okay to leave as is. 

```suggestion
sidebar_label: Consensus
```